HardwareObjects/BlissEnergy.py

```
@@ -106,7 +109,8 @@ def set_value(self, value, timeout=0):
             ValueError: Value not valid or attemp to set write only actuator.
```

I have problems here:
1)  the documentatono still says that you will raise ValueError if you try to write to a read-only actuator.
2) If I set a HardwareObject value to say 10, I expect that the value will henceforth be 10, or that I get an error or otherwise be noticed that something went amiss. I wold not expect that the value remains at 7 and my command is just silently ignored. Likely there is a reason for this change, but I wonder what it is

---

Hm I somehow agree, The code that uses this however did not at all handle ValueError so I think this was the easiest fix. I don't think we saw any problem with it returning silently. We can go through the code again and handle the ValueError if its really a problem.

---

Arguably it is an error to try to set the energy on a fixed-energy beamline. Well, as long as this is only called from this context or by people who understand hte behaviour, there will be no problem. If some outsider (like me) writes code that  sets the energy it would be a lot easier for him to find his mistake if he got an error, rather than letting his code proceed on the assumption that the eneergy had been changed.

---

Hm, I think its good of we can keep the ValueError in general for actuators and I will investigate where we need to change to also have it for BlissEnergy. I suggest that we keep it like this for the moment as we have tested with this. Ill make a comment in the code that we need to take another look at handling the ValueError, and Ill try to test that the coming weeks. How about that ?

---

Sounds fine.